038-MdwCovid19/README.md

```diff
@@ -0,0 +1,65 @@
+
+# Modern Data Warehouse - COVID 19
+## Introduction
+This hack is designed so that the students will construct a fully functional Modern Data Pipeline utilizing COVID-19 data and then creating calcuations on growth vs policies enacted by different governments.  The process of collecting, organizing and making inferences based on different data sources is someting that most data practioners need to do at some point in there career.  This hack teaches how to do this in Azure.  
+**Note:  This lab is recommended to be done over at least three days.  Is it very in-depth and will test most students.**
+
+## Learning Objectives
+In this hack you will be working to make a recommendation to a fictional government on the COVID-19 mitigation policies they should enact based on collecting, cleaning, correlating and examining avilable data sets. 
+
+The technical learning objectives:
+
+1. Provision a Data Lake
+2. Land data in the Data Lake from Cloud resources (Relational and CosmosDB).
+3. Land data in the Data Lake from On-Premise resources (an Azure VM is used to simulate an on-prem store).
+4. Create Data Pipelines to merge the datasets into usuable format.
+5. Define Star Schemas and create a Data Warehouse.
+6. Enact version control and administrive approval for all pull requests within Github.
+7. Perform calculations on Fact tables.
+8. Enable Unit Tests
+
+## Challenges
+- Background: **[Learn about the fictional government of Caladan and your tasks](Student/00-Background.md)**
+   - Prepare for the challenge
+- Challenge 1: **[Provision](Student/01-Provision.md)**
+   - Provision a Data Storage solution for landing your COVID-19 data sources.
+- Challenge 2: **[Ingest 1](Student/02-CloudIngest.md)**
+   - Ingest COVID-19 data sources from cloud resources.
+- Challenge 3: **[Ingest 2](Student/03-OnPremIngest.md)**
+   - Ingest COVID-19 data sources from On-Premises resources.
+- Challenge 4: **[Transform and Load](Student/04-TransformLoad.md)**
+   - Configure data pipelines to transform and load the data into usable sources for a Data Warehouse.
+- Challenge 5: **[Calculate](Student/05-Calculate.md)**
+   - Create Data Warehouse and calculations for growth vs policy. 
+
+## Prerequisites
+- Your own Azure subscription with Owner access and follow the instruction of the **[Lab Deployment](036-MDW-COVID19\Coach\00-PreReqs.md)**
```

Comment:
Couple of comments here:
1.) Link to "Lab Deployment" is broken.  It's not using a relative path and was hard coded to the old number.
2.) Even if the link is fixed, it would be linking directly to the new 00-PreReqs.md in the Coach's guide!  It's a direct path for the attendees to the answers.

I can't remember what we decided...  is this Lab Deployment designed to be completed a) by a coach ahead of time?  Or b) by the attendees themselves?

If A, then there should be NO link here on the homepage. Just the statement of permissions required for the hack.
If B, then we need to re-think these instructions being part of the STUDENT pre-reqs/challenge 0 instead of the Coach's guide.